mix.exs

```diff
@@ -1,7 +1,7 @@
 defmodule Membrane.MP4.Plugin.MixProject do
   use Mix.Project
 
-  @version "0.18.0"
+  @version "0.18.1"
```

Comment:
Change of the h264 format used changes the API - shouldn't we bump the minor?

Reply:
But does it really? `0.5.0` just made one field optional, so for `0.4.0`'s users who explicitly provide the value, it shouldn't matter.